### PR TITLE
Prepare v0.2.3 release

### PR DESCRIPTION
**Pull request by dhimmel**
Follows up on https://github.com/manubot/manubot/pull/96.

Adds release notes.
Travis CI: update encrypted PyPI deploy password

### CODE REVIEW

File: release-notes/v0.2.3.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+------------
+
+Citations of shortDOIs are now supported (see [GH92](https://github.com/manubot/manubot/issues/92) and [GH93](https://github.com/manubot/manubot/pull/93)).
+[shortDOIs](http://shortdoi.org/), which start with `10/` rather than `10.`, can now be cited just like a DOI. For example, `@doi:10/gddkhn` is a supported citation.
+Manubot expands shortDOI citations to their regular DOIs, e.g. `@doi:10.1098/rsif.2017.0387`, such that `manubot process` will treat both the short and regular form as the same citation.
+
+Bug fixes
+---------
+
+Queries to Manubot's tanslation-server now specify `single=1` to enforce returning a single record per persistent identifier (see [GH90](https://github.com/manubot/manubot/pull/90)).
```
Comment (agitter) [inline release-notes/v0.2.3.md]:
```suggestion
Queries to Manubot's translation-server now specify `single=1` to enforce returning a single record per persistent identifier (see [GH90](https://github.com/manubot/manubot/pull/90)).
```

File: release-notes/v0.2.3.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Previously, multiple results were [sometimes](https://github.com/zotero/translation-server/issues/65) returned, causing Manubot's JSON CSL retrieval to fail.
+Furthermore, Zotero child notes now ignored, fixing another [failure mode](https://github.com/zotero/translation-server/issues/67) for CSL export of Zotero metadata.
```
Comment (agitter) [inline release-notes/v0.2.3.md]:
```suggestion
Furthermore, Zotero child notes are now ignored, fixing another [failure mode](https://github.com/zotero/translation-server/issues/67) for CSL export of Zotero metadata.
```

**Comment by dhimmel** [comment]
> If we want to start automatically archiving releases on Zenodo or figshare, this would be the time to decide.

As per https://github.com/greenelab/meta-review/issues/135#issuecomment-462179156, i'd say let's release 0.2.3 without Zenodo / Figshare archiving.

**Comment by dhimmel** [comment]
I pushed the `v0.2.3` tag after merging. PyPI deployment failed according to the [Travis log](https://travis-ci.com/manubot/manubot/jobs/176983327#L1104):

```
Could not restore untracked files from stash entry
PyPI upload failed.
failed to deploy
```

Similar issue it seems to https://github.com/FaradayRF/faradayio/issues/35. Will investigate and attempt to correct. May have to make some commits editing `.travis.yml`. May just keep amending latest commit till it works.

**Comment by dhimmel** [comment]
Regenerating the encrypted password for PyPI deployment on Travis in 81394c0a8153fa3ad41ee3cb987227c8ea043125 fixed it. 

https://pypi.org/project/manubot/0.2.3/ is now up!

**Comment by agitter** [comment]
That's great that the deployment is now set up for `manubot/manubot`.

I wonder what's going on with the AppVeyor builds.  It looks like it ran on 81394c0a8153fa3ad41ee3cb987227c8ea043125 twice, once on master and once for the tag.  The master [build failed](https://ci.appveyor.com/project/manubot/manubot/builds/22296213) so the PyPI badge shows a failed build, but the [tag build succeeded](https://ci.appveyor.com/project/manubot/manubot/builds/22296223).

**Comment by dhimmel** [comment]
> I wonder what's going on with the AppVeyor builds. It looks like it ran on [81394c0](https://github.com/manubot/manubot/commit/81394c0a8153fa3ad41ee3cb987227c8ea043125) twice, once on master and once for the tag.

This is expected and also occurs on Travis. On Travis we need the build to run twice, since only the tag build is set to deploy.

The failures were because some CSL JSON generated from translation-server had an unexpected title. `pandas-dev/pandas` rather than `Flexible and powerful data analysis`. If this starts to become a frequent issue I'll try to track down the cause. Perhaps some aspect of the webpage didn't load fully or there is some non-deterministic behavior in translation-server.

**Pull request merged**